tecken/upload/utils.py

```diff
@@ -153,6 +153,9 @@ def __init__(self, path, name):
     def size(self):
         return os.stat(self.path).st_size
 
+    def __repr__(self):
```

Comment:
This is a function that runs whenever you print the repr of something. It's better than the default which in this case is something like `<FileMember object at 0x7fa4879902e0>`. Now it'll print something like `<FileMember /tmp/uploads/ou812/something.sym something.sym>`.

Reply:
TIL: https://docs.python.org/3/library/functions.html#repr

Reply:
repr is really handy sometimes. It's what gets called when you do things like:

```
logger.info("watchdir: %r", watchdir)

print(f"watchdir: {watchdir!r}")
```